roles/vgpu/tasks/configure.yml

```diff
@@ -13,15 +13,6 @@
     file: configure-mig.yml
   when: vgpu_mig_enabled | bool
 
-- name: Template nvidia-mdev service
```

Comment:
Changing this from a systemd templated unit file to a regular unit seems to work without having to manually remove the old files:
```
[root@juno ~]# mv /etc/systemd/system/test@{hello.service,hello2.service}
[root@juno ~]# sudo systemctl daemon-reload 
[root@juno ~]# systemctl enable test@hello.service
Created symlink /etc/systemd/system/multi-user.target.wants/test@hello.service → /etc/systemd/system/test@.service.
[root@juno ~]# mv /etc/systemd/system/test@{hello2.service,hello.service}
[root@juno ~]# sudo systemctl daemon-reload 
[root@juno ~]# systemctl enable test@hello.service
Removed "/etc/systemd/system/multi-user.target.wants/test@hello.service".
Created symlink /etc/systemd/system/multi-user.target.wants/test@hello.service → /etc/systemd/system/test@hello.service
```